components/SocialMediaConnect.vue

```diff
@@ -135,6 +150,15 @@ export default {
           break;
       }
     },
+    openURL(socialMedia) {
+      switch (socialMedia.id) {
+        case 'facebook':
+          openURL(this, `https://facebook.com/${this.user.facebookId}`);
```

Comment:
cannot open URL like this, blocked by facebook
can amend according to https://github.com/likecoin/likecoin-store/pull/614?